dotCMS/src/integration-test/java/com/dotcms/rest/api/v2/contenttype/FieldResourceTest.java

```diff
@@ -82,6 +82,75 @@ public static void cleanUpData() throws DotDataException, DotSecurityException {
         APILocator.getContentTypeAPI(APILocator.systemUser()).delete(contentType);
     }
 
+    @Test
+    public void testCreateRelationshipFieldWithDash_Return400() throws Exception {
+        final FieldResource resource = new FieldResource();
+
+        final ContentType contentType = getContentType();
+
+        final String jsonField = "{"+
+                // IDENTITY VALUES
+                "	\"clazz\" : \"com.dotcms.contenttype.model.field.ImmutableRelationshipField\","+
```

Comment:
this hardcoded value can be replaced by `ImmutableRelationshipField.class.getCanonicalName()`

Reply:
done